components/Board/Board.jsx

```diff
@@ -0,0 +1,17 @@
+import styles from '@/styles/Board.module.css'
+import Card from '@/components/Board/Card'
+
+const Board = (props) => {
+    const { cards, title } = props
+    return (
+        <div className={styles.container}>
+            <h3 className={styles.title}>{title}</h3>
+            <div className={styles.board_box}>
+                {cards.map((card) => (
```

Comment:
Please add error boundaries (i.e.: when there's no array). You can use a question mark on the map function)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Done